src/main/java/core/basesyntax/service/impl/ReturnOperation.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperationHandler;
+
+public class ReturnOperation implements FruitOperationHandler {
+    @Override
+    public int apply(FruitRecordDto fruitRecordDto) {
+        Fruit fruit = new Fruit(fruitRecordDto.getFruitName());
+        Integer currentQuantity = Storage.fruits.get(fruit);
+        int returnAmount = fruitRecordDto.getQuantity();
+        if (returnAmount < 0) {
+            throw new RuntimeException("Return amount can't be less than 0");
+        }
+        int newQuantity = currentQuantity + returnAmount;
```

Comment:
let's make this code safer. Now you can get NPE here because Storage.fruits.get(fruit) can return null